services/wallet/controllers_v3_pvt_test.go

```diff
@@ -1,4 +1,4 @@
-package wallet_test
```

Comment:
moved this to wallet this is distinct from integration tests in the wallet_test package. Will eventually cleanup this up and separate out  integration tests etc.